File: 백준/Bronze/5622. 다이얼/다이얼.py
eng = ['ABC', 'DEF', 'GHI', 'JKL', 'MNO', 'PQRS', 'TUV', 'WXYZ']
word = input()
time = int(0)
for i in eng:
  for j in i:
    for h in word:
      if j == h:
        time += eng.index(i) + 3
print(time)

File: C++17/백준/Bronze/5622. 다이얼/다이얼.cc
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int main() {
	ios_base::sync_with_stdio(false);
	cin.tie(NULL);

    string grandma_num;
    vector<string> call_num = {"ABC","DEF","GHI","JKL","MNO","PQRS","TUV","WXYZ"};
    int time = 0;
    cin >> grandma_num;
    for (char c : grandma_num){
        for (int j = 0; j<8; j++){
            for (char d : call_num[j]){
                if (c == d) {
                    time += j+3;
                    break;
                }
            }
        }
    }

    cout << time;

    return 0;
} 